plugin-api/src/main/java/org/hyperledger/besu/plugin/data/Transaction.java

```diff
@@ -147,4 +170,18 @@
    * @return the transaction payload
    */
   Bytes getPayload();
+
+  /**
+   * Returns whether or not the transaction is a legacy transaction.
+   *
+   * @return true if legacy transaction, false otherwise
+   */
+  boolean isLegacyTransaction();
```

Comment:
Should be a default method and `@Unstable`

Reply:
Yeah

Reply:
Done